### ISSUE TITLE
Tensorboard Loss Graph Acting Strangely

### ISSUE THREAD

**Issue by Spawnfile** [milesial/Pytorch-UNet#246]
Hello,

While in training phase Criterion loss goes down nicely but goes to the peak value after that. Accordingly I can not deduce a outcome from this loss values about model accuracy.

Here an example of this stuff. My question is how can I follow up the accuracy when I make some changes in the model when this loss thing is happening ?    

Greetings.

![Screenshot from 2020-12-02 13-37-48](https://user-images.githubusercontent.com/31824731/102742640-71ce4200-4366-11eb-8d10-37902def28cd.png)

**Comment by milesial** [comment]
Your learning rate may be way too high. Try dividing it by 10, 100, ... and search the best parameter for your task

**Comment by Spawnfile** [comment]
`lr` is 0.001, default value but I will try, thank you.
